src/randomstring.ts

```diff
@@ -15,10 +15,21 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+import { encodeUnpaddedBase64Url } from "./base64";
+import { crypto } from "./crypto/crypto";
+
 const LOWERCASE = "abcdefghijklmnopqrstuvwxyz";
 const UPPERCASE = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
 const DIGITS = "0123456789";
 
+export function secureRandomBase64(len: number): string {
+    const key = new Uint8Array(len);
+    crypto.getRandomValues(key);
+
+    // encode to base64url as this value goes into URLs
```

Comment:
It seems a little odd that this function should know this fact about its usage. Perhaps it should be named `secureRandomBase64Url`?